paywall/src/components/lock/UnlockFlag.js

```diff
@@ -26,7 +28,10 @@ export const UnlockedFlag = () => {
   return (
     <Flag id="UnlockFlag" hidden={hidden}>
       <RoundedLogo size="28px" />
-      <p>Subscribed with Unlock</p>
+      <p>
+        Subscribed with&nbsp;
+        <a href="https://paywall.unlock-protocol.com">Unlock</a>
```

Comment:
Same as above? I worry this will break on local dev and/or staging and be hard to change when/if we change domains?

Reply:
Definitely open to using config.paywallUrl here, does that make sense to you too?

Reply:
Yes